src/main/java/cj/software/genetics/schedule/client/javafx/EditSchedulingProblemController.java

```diff
@@ -74,17 +76,36 @@ public class EditSchedulingProblemController implements Initializable {
     @FXML
     private TextField tfWorkersCount;
 
+    @FXML
+    private TextField tfElitismCount;
+
+    @FXML
+    private TextField tfTournamentSize;
+
+    @FXML
+    private Spinner<Double> spMutationRate;
+
     private SchedulingProblemUiModel model;
 
     public void setModel(SchedulingProblemUiModel model) {
         this.model = model;
         tblPriorities.setItems(model.getPriorities());
         Bindings.bindBidirectional(tfSolutionsCount.textProperty(), model.solutionCountProperty(), numberStringConverter);
         Bindings.bindBidirectional(tfWorkersCount.textProperty(), model.workerCountProperty(), numberStringConverter);
+        Bindings.bindBidirectional(tfElitismCount.textProperty(), model.elitismCountProperty(), numberStringConverter);
+        Bindings.bindBidirectional(tfTournamentSize.textProperty(), model.tournamentSizeProperty(), numberStringConverter);
+        spMutationRate.getValueFactory().valueProperty().bindBidirectional(model.mutationRateProperty().asObject());
```

Comment:
Is this line perhaps not necessary because of the bug mentioned in method `getModel()`?